prometheus/Dockerfile.grok-exporter

```diff
@@ -0,0 +1,20 @@
+# target: libvineyard/vineyardd-grok-exporter:latest
+#
+FROM frolvlad/alpine-glibc:alpine-3.12
```

Comment:
/cc @univerone I have updated the docker image to use alpine image as the base image, FYI.